view/src/service.rs

```diff
@@ -211,6 +212,16 @@ impl ViewService {
             catching_up,
         })
     }
+
+    #[instrument(skip(self))]
+    pub async fn await_change(
```

Comment:
I think we could probably elide this 
method and pass directly from the proto generated trait to the method on the underlying storage